app/models/edition/national_applicability.rb

```diff
@@ -12,10 +12,16 @@ def process_associations_before_save(edition)
   included do
     has_many :nation_inapplicabilities, foreign_key: :edition_id, dependent: :destroy, autosave: true
     validates_associated :nation_inapplicabilities
+    alias_method :excluded_nations, :nation_inapplicabilities
+    validates :excluded_nations, length: { maximum: Nation.all.count - 1, message: 'can not exclude all nations' }
```

Comment:
If this validation isn't on the `nation_inapplicabilities` attribute is it going to be transferred to the view?

Reply:
This adds an alert at the top of the page with the desired text, will add a screenshot for clarity

Reply:
have changed this to use translations to get the wording I want